src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,111 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
```

Comment:
At least one empty constructor should exist. Because JPA need it for mapping the entities.